build-farm/platform-specific-configurations/linux.sh

```diff
@@ -68,6 +68,9 @@ then
     then
       export CONFIGURE_ARGS_FOR_ANY_PLATFORM="${CONFIGURE_ARGS_FOR_ANY_PLATFORM} --enable-cuda --with-cuda=$CUDA_HOME"
     fi
+  elif [ "${ARCHITECTURE}" == "aarch64" ] && [ -f /usr/local/cuda-10.2/include/cuda.h ]; then
```

Comment:
Is that 10.2 detection too brittle?

Reply:
Possibly, although I installed 10.2 on my machine pre-lockdown and as far as I can see it's still that version ;-)

Reply:
(This was part of a different PR and should not have been included in this one - resolving)